SKYKitChat/Classes/UI/ConversationView/SKYChatConversationViewController.swift

```diff
@@ -342,6 +370,46 @@ extension SKYChatConversationViewController {
 
         self.skygear.chatExtension?.subscribeToMessages(in: self.conversation!,
                                                         handler: self.messageChangeHandler!)
+
+    }
+
+    open func subscribeTypingIndicatorChanges() {
+        if self.typingIndicatorChangeHandler == nil {
+            self.typingIndicatorChangeHandler = {(indicator) in
```

Comment:
Same as [the comment above](https://github.com/SkygearIO/chat-SDK-iOS/pull/18#discussion_r99985409).